src/lib/operators/get_table.cpp

```diff
@@ -239,8 +239,38 @@ std::shared_ptr<const Table> GetTable::_on_execute() {
     ++output_chunks_iter;
   }
 
+  // Check if a table index is irrelevant because of pruning.
+  const auto check_if_index_is_irrelevant_lambda = [this](const std::shared_ptr<AbstractTableIndex>& table_index) {
```

Comment:
We usually don't put the name of the used language construct (here: lambda) into the variable name.

Furthermore, I'd try to be even more descriptive here: the index might be irrelevant because the column id might be pruned. How about creating a lambda function `is_indexed_column_pruned` or something similar?